Gopkg.toml

```diff
@@ -52,7 +52,7 @@
   name = "github.com/korovkin/limiter"
 
 [[constraint]]
-  branch = "master"
+  branch = "feature/analyzer-recognizers-endpoint"
```

Comment:
needs to change before merging.
you can already change back so you wont forget

Reply:
fixed